content/en/real_user_monitoring/mobile_and_tv_monitoring/ios/advanced_configuration.md

```diff
@@ -654,6 +655,7 @@ Also, you can configure first party hosts using `urlSessionTracking`. This class
 
 For instance, you can configure `example.com` as the first party host and enable both RUM and Tracing features:
 
+[10]: https://developer.apple.com/documentation/foundation/urlsession/init(configuration:delegate:delegatequeue:)#parameters
```

Comment:
It looks like this reference link isn't used anywhere. If that's expected, this can be removed from here and from line `970`.